mapproxy/test/system/test_mapnik.py

```diff
@@ -57,8 +57,28 @@
 </Map>
 """.strip()
 
-def setup_module():
+mapnik_fonts_xml = b"""
+<?xml version="1.0"?>
+<!DOCTYPE Map>
+<Map background-color="#ff0000" bgcolor="#ff0000" srs="+proj=latlong +datum=WGS84">
+    <Layer name="marker">
+        <StyleName>marker</StyleName>
+        <Datasource>
+            <Parameter name="type">ogr</Parameter>
+            <Parameter name="file">test_point.geojson</Parameter>
+            <Parameter name="layer">OGRGeoJSON</Parameter>
+        </Datasource>
+    </Layer>
+    <Style name="marker">
+        <Rule>
+            <TextSymbolizer face-name="Fira Sans Regular">[name]</TextSymbolizer>
```

Comment:
We already bundle a font for the attribution lines in `mapproxy/image/fonts`. Can you use this one?